docs/intro/quickstart/setup.rst

```diff
@@ -0,0 +1,69 @@
+.. _ref_quickstart_setup:
+
+===========================
+Setting up your environment
+===========================
+
+.. edb:split-section::
+
+  We will clone our Next.js starter template into a new directory called ``flashcards``. This will create a fully configured Next.js project and a local Gel instance with an empty schema. You should see the test suite pass, indicating that the database instance was created successfully, and we're ready to start building our application.
+
+  .. code-block:: sh
+
+      $ git clone \
+          git@github.com:geldata/quickstart-nextjs.git \
+          flashcards
+      $ cd flashcards
+      $ npm install
+      $ npm run test
+
+
+.. edb:split-section::
+
+  Let's quickly take a poke around the empty database with our CLI REPL.
+
+  .. code-block:: sh
+
+      $ npx gel
```

Comment:
Wait, we haven't run `npx gel project init`, so `npx gel` will just fail as it won't be able to connect to anything

Reply:
Oh right, this was originally starting with the `npm create @gel` flow which _does_ the `project init` flow for you. I'll add it and add some explanation.